aio/content/examples/getting-started/src/app/cart/cart.component.ts

```diff
@@ -24,14 +24,15 @@ export class CartComponent {
     private formBuilder: FormBuilder,
   ) {
 // #enddocregion inject-form-builder
-    this.items = this.cartService.getItems();
-
     this.checkoutForm = this.formBuilder.group({
       name: '',
       address: ''
     });
 // #docregion inject-form-builder
   }
+  ngOnInit() {
```

Comment:
Please, leave an empty line between constructor/methods:

```suggestion

  ngOnInit() {
```